qlib/data/storage/file_storage.py

```diff
@@ -26,7 +26,7 @@ class FileStorageMixin:
 
     @property
     def dpm(self):
-        return C.DataPathManager(self.provider_uri, None)
+        return C.DataPathManager(self.provider_uri, C.mount_path)
```

Comment:
There may be conflictions when merging with https://github.com/microsoft/qlib/pull/784
Please help to review #784  and then let's merge #784  and this PR in order